plugin/package.json

```
@@ -12,9 +12,9 @@
     "build:dev": "yarn clean && sh -ac '. ./.env.development; yarn ts-node node_modules/.bin/webpack'",
     "start": "sh -ac '. ./.env.development; yarn ts-node node_modules/.bin/webpack serve'",
     "start-console": "./start-console.sh",
-    "cypress": "cypress open -e TAGS=\"@ossmc and @core and not @skip-ossmc\"",
-    "cypress:run": "cypress run -e TAGS=\"@ossmc and @smoke\" && cypress run -e TAGS=\"@ossmc and @core and not @smoke and not @skip-ossmc\" && cypress run -e TAGS=\"@ossmc and @crd-validation\"",
-    "cypress:run:junit": "cypress run --reporter cypress-multi-reporters --reporter-options configFile=reporter-config.json -e TAGS=\"@ossmc and @core and not @skip-ossmc\" && cypress run --reporter cypress-multi-reporters --reporter-options configFile=reporter-config.json -e TAGS=\"@ossmc and @crd-validation\"",
+    "cypress": "cypress open -e TAGS=\"@ossmc and @core1 and @core2 and not @skip-ossmc\"",
+    "cypress:run": "cypress run -e TAGS=\"@ossmc and @smoke\" && cypress run -e TAGS=\"@ossmc and @core1 and @core2 and not @smoke and not @skip-ossmc\" && cypress run -e TAGS=\"@ossmc and @crd-validation\"",
+    "cypress:run:junit": "cypress run --reporter cypress-multi-reporters --reporter-options configFile=reporter-config.json -e TAGS=\"@ossmc and @core1 and @core2 and not @skip-ossmc\" && cypress run --reporter cypress-multi-reporters --reporter-options configFile=reporter-config.json -e TAGS=\"@ossmc and @crd-validation\"",
```

```suggestion
    "cypress": "cypress open -e TAGS=\"@ossmc and (@core-1 or @core-2) and not @skip-ossmc\"",
    "cypress:run": "cypress run -e TAGS=\"@ossmc and @smoke\" && cypress run -e TAGS=\"@ossmc and (@core-1 or @core-2) and not @smoke and not @skip-ossmc\" && cypress run -e TAGS=\"@ossmc and @crd-validation\"",
    "cypress:run:junit": "cypress run --reporter cypress-multi-reporters --reporter-options configFile=reporter-config.json -e TAGS=\"@ossmc and (@core-1 or @core-2) and not @skip-ossmc\" && cypress run --reporter cypress-multi-reporters --reporter-options configFile=reporter-config.json -e TAGS=\"@ossmc and @crd-validation\"",
```

No test will have `@core-1` and `@core-2`. If you want to run them all then I think you need an `or` condition.